packages/calendar-input/src/types.ts

```diff
@@ -7,11 +7,11 @@ export type StyledCalendarInputProps = {
 export type CalendarInputProps = {
   popoverAppearance?: CalendarPopoverAppearanceType
   inputAppearance?: TextInputAppearanceType
-  onChange?: (date?: Date) => void
+  onChange?: (date?: string) => void
   height?: number
   width?: number
   className?: string
-  value?: Date
+  value?: string
```

Comment:
Would be cool if value could be passed in `string`, `Date` and `Dayjs` format. Underneath you can use dayjs to operate on date. onChange should be called with value in format that it was originally passed - if value is string then onChange(value as string), if Date then onChange(value as Date) etc

Reply:
I wanted to do this way but what when a value is empty? Calendar doesn't know date format, so probably it should be done with a `valueFormat` prop?